app/models/cafe_work.rb

```diff
@@ -97,23 +95,23 @@ def add_worker(worker_params, user)
   # /d.wessman
   def update_worker(worker_params, user)
     if !owner?(user)
-      errors.add('Auktorisering',
-                 'misslyckades, du har inte rättighet att redigera.')
+      errors.add(I18n.t('authorization'),
```

Comment:
http://stackoverflow.com/questions/21161640/access-translation-file-i18n-from-inside-rails-model
`t` is a ´view_helper` and not accessible within models.